sblecdemo/src/main/java/com/nexenio/sblecdemo/DemoLayout.java

```diff
@@ -190,8 +193,15 @@ public void hideReceivingFailedError() {
 
     @NonNull
     @Override
-    public Context getContext() {
-        return this;
+    public Activity getActivity() {
+        Context context = getContext();
+        while (context instanceof ContextWrapper) {
```

Comment:
Can you please explain, why a while loop is needed here?

Reply:
Because you can't know how deep the current view is nested in the hosting activity (there could be a fragment in between, for example). See https://stackoverflow.com/a/32973351/1188330